Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -165,6 +178,32 @@ void DMainWindow::OnOpen()
 		StartGame(filename);
 }
 
+void DMainWindow::OnBrowse()
+{
+	std::string path = ShowFolderDialog().toStdString();
+	std::vector<std::string>* iso_folder = &SConfig::GetInstance().m_ISOFolder;
+	if (!path.empty()){
```

Comment:
Opening brace on next line please.
And I personally think that the next line is one of those where using ```auto``` makes sense (but someone else might disagree here).